### ISSUE TITLE
Add a field to the net_certificate table to specify a port

### ISSUE THREAD

**Issue by bdd4329** [turbot/steampipe-plugin-net#48]
**Is your feature request related to a problem? Please describe.**
I would like to be able to monitor certificate expiration dates on other commonly used TLS ports such as 636, 8443, etc. Using an alternative product to monitor other ports means that I can't easily have a single dashboard for all of my certs if I want to use Steampipe.

**Describe the solution you'd like**
"port" should be added as a field to the net_certificate table, which would enable certificates on arbitrary ports to be monitored. I assume that it's not there right now so that it doesn't need to be specified for most cases.

**Describe alternatives you've considered**
Use an alternative product to monitor these ports.

**Additional context**
None

**Comment by bdd4329** [comment]
I can see that @cbruno10 added a PR for this issue (#50). Is there some reason why his PR hasn't been approved?

**Comment by cbruno10** [comment]
Hi @bdd4329 , for that PR, I had started work on it, but then had a thought that instead of just adding `port`, what if instead we added a new column that would take a full address, including the port (similar to other tables in this plugin), and deprecate the current `domain` column.

I was assessing the impact of this change on the table + mods that use, like the [Net Insights Mod](https://hub.steampipe.io/mods/turbot/net_insights), and then unfortunately got distracted with some other priorities.

I'll take another look at the PR and its impact again this week, but in the meantime @bdd4329 , if this is a blocker for you, feel free to locally compile the `add-certificate-port` branch and see if the new `address` column helps with your requirements.
